addons/sale_loyalty/i18n/sale_loyalty.pot

```diff
@@ -431,6 +431,13 @@ msgstr ""
 msgid "This coupon is expired."
 msgstr ""
 
+#. module: sale_loyalty
+#. odoo-python
+#: code:addons/sale_loyalty/models/sale_order.py:0
+#, python-format
+msgid "This program cannot be applied with code."
+msgstr ""
+
```

Comment:
you can drop the pot update in master, it'll be updated after freeze when the translations are exported ;).